src/20250421_Multi_GHOGnosisLaunch/GHOGnosisLaunch.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "GHO Gnosis Launch"
+author: "kpk"
+discussions: "https://governance.aave.com/t/arfc-launch-gho-on-gnosis-chain/21379"
+---
+
+## Simple Summary
+
+This AIP proposes the expansion of GHO, the native asset of the Aave Protocol, to the Gnosis Chain network utilizing the Chainlink Cross-Chain Interoperability Protocol (CCIP). This follows the consensus reached in the ARFC phase initiated by kpk.
+
+The necessary smart contracts and configurations have been developed and reviewed to ensure a secure deployment on Gnosis Chain.
+
+## Motivation
+
+Building on GHO's presence on other networks, expanding to Gnosis Chain offers significant advantages. Gnosis is known for its low transaction fees, an engaged community, and a rapidly growing suite of DeFi applications, particularly focused on Real World Assets (RWAs) and innovative payment solutions like Gnosis Pay. This expansion aims to increase GHO's utility, provide the foundation for sGHO deployment on Gnosis, and ultimately enhance Aave's Total Value Locked (TVL) and revenue on this chain.
+
+## Specification
+
+This AIP includes a series of actions required to launch GHO on Gnosis Chain:
+
+1.  Configure new Chainlink CCIP lanes between the chains GHO is listed with a rate limit capacity of \*1,500,000 GHO** and **300 GHO per second\*\* refill rate. An Ethereum-based GHO Facilitator with a specified mint cap of 20M GHO will support this bridge.
+2.  Configure and activate GhoAaveSteward and GhoCcipSteward on Gnosis Chain to control GHO listing parameters and the CCIP lane configuration.
```

Comment:
Missing GhoBucketSteward here. Also, do we wanna add the risk council address used?